src/views/PackageDetail.vue

```diff
@@ -13,12 +13,13 @@
       </div>
       <div class="hero-body">
         <div class="container">
+          <router-link to="/gallery" class="back-arrow title">
```

Comment:
Thanks for making these changes! Final thing: I would prefer the header of this page to be how it was originally formatted, without the description.

The reason is that the description field is redundant on when it's used on module READMEs as well.

Example: 
![image](https://user-images.githubusercontent.com/20846869/94042177-3c32b080-fd99-11ea-8e12-3520b177f289.png)